docs/id/docs/index.md

```diff
@@ -22,27 +22,27 @@
 
 ---
 
-**Documentation**: <a href="https://fastapi.tiangolo.com" target="_blank">https://fastapi.tiangolo.com</a>
+**Documentation**: <a href="https://fastapi.tiangolo.com/id" target="_blank">https://fastapi.tiangolo.com/id</a>
 
 **Source Code**: <a href="https://github.com/tiangolo/fastapi" target="_blank">https://github.com/tiangolo/fastapi</a>
 
 ---
 
-FastAPI is a modern, fast (high-performance), web framework for building APIs with Python 3.6+ based on standard Python type hints.
+FastAPI adalah web framework untuk membangun beragam API dengan menggunakan Python 3.6+ berdasarkan Python type hints yang baku.
 
-The key features are:
+Fitur-fitur yang tersedia antara lain:
 
-* **Fast**: Very high performance, on par with **NodeJS** and **Go** (thanks to Starlette and Pydantic). [One of the fastest Python frameworks available](#performance).
+* **Fast**: Memiliki performa yang sangat tinggi,  **NodeJS** and **Go** (terima kasih untuk Starlette dan Pydantic). [Salah satu dari framework python tercepat yang ada](#performance).
```

Comment:
suggestion for this line  35:

* **Fast**: Memiliki performa yang sangat tinggi,  setara dengan **NodeJS** dan **Go** (terima kasih untuk Starlette dan Pydantic). [Salah satu dari framework python tercepat yang ada](#performance).